src/core/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,85 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { useTheme } from '../utils/hooks/useTheme';
+import '@itwin/itwinui-css/css/side-navigation.css';
+import { CommonProps } from '../utils/props';
+import SvgChevronRight from '@itwin/itwinui-icons-react/cjs/icons/ChevronRight';
+import { IconButton } from '../Buttons';
+
+export type SidebarProps = {
+  /**
+   * Buttons shown in the top portion sidebar.
+   * Recommended to use `SidenavButton`s.
```

Comment:
Apostrophe looks weird.. Maybe just 'SidenavButton' components.

Reply:
What apostrophe? 😄 
![image](https://user-images.githubusercontent.com/9084735/118654215-53a22c00-b7b6-11eb-9ac7-86423fbee285.png)

Reply:
well I mean.. it is written in there, but anyway.. point being, looks bad :D

Reply:
I pushed a change earlier